server/src/main/java/org/opensearch/search/aggregations/bucket/FastFilterRewriteHelper.java

```diff
@@ -80,13 +90,13 @@ private static Query unwrapIntoConcreteQuery(Query query) {
     }
 
     /**
-     * Finds the min and max bounds of field values for the shard
+     * Finds the global min and max bounds of the field for the shard from each segment
```

Comment:
The "global" and "from each segment" parts are a bit confusing. Maybe "across all segments"?

Reply:
done